modules/administration-guide/partials/ref_grafana-dashboards-for-the-dev-workspace-operator.adoc

```diff
@@ -10,30 +12,19 @@ image::monitoring/monitoring-dev-workspace-metrics-panel.png[Grafana dashboard p
 
 Average workspace start time:: The average workspace startup duration.
 Workspace starts:: The number of successful and failed workspace startups.
-Workspace startup duration:: A heatmap that displays workspace startup duration.
 {devworkspace} successes / failures:: A comparison between successful and failed {devworkspace} startups.
 {devworkspace} failure rate:: The ratio between the number of failed workspace startups and the number of total workspace startups.
 {devworkspace} startup failure reasons:: A pie chart that displays the distribution of workspace startup failures:
 * `BadRequest`
 * `InfrastructureFailure`
 * `Unknown`
 
-== The *Operator metrics* panel (part 1)
+== The *Operator metrics* panel
```

Comment:
The UI label in the screenshot has both Operator and Metrics in capitals, hence my suggestion:
```suggestion
== The *Operator Metrics* panel
```